frontend/src/components/home-page/content-tree/content-tree.tsx

```diff
@@ -322,7 +326,7 @@ const ContentTree = (): ReactElement => {
                                                 }
                                               >
                                                 <div className="title">
-                                                  {question.title}
+                                                  {truncate(question.title)}
```

Comment:
add
```
    text-overflow: ellipsis;
    display: -webkit-box;
    -webkit-line-clamp: 2;
    -webkit-box-orient: vertical;
```
to the `title` class
```
  .title,
  .type {
    overflow: hidden;
    font-size: 1.25rem;
  }
```
to have ellipsis replace text longer than 2 lines.